lib/components/index.ts

```diff
@@ -17,6 +17,7 @@ export { Dropdown } from './Dropdown';
 export { FitText } from './FitText';
 export { Flex } from './Flex';
 export { Floating } from './Floating';
+export { useFuzzySearch } from './FuzzySearch';
```

Comment:
It's not really a component, it's a hook; as such I think it should go in common.

I also dislike that we're using a barrel file, so having it in common prevents adding more to the "problem"